packages/lsp-server/package/test/server.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { expect } from "chai";
+import {
+  TextDocument,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position
+} from "vscode-languageserver";
+
+import { validateDocument, SYNTAX_ERROR_MSG } from "../src/language-services";
+
+describe("the XML Language Server", () => {
```

Comment:
because of previous comment (file name)
`the XML Language Server` -->  `the XML Language Services`